mantle/cosa/schema.go

```diff
@@ -0,0 +1,74 @@
+package cosa
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+
+	"github.com/pkg/errors"
+
+	schema "github.com/xeipuuv/gojsonschema"
+)
+
+var (
+	// JSON Schema document. Default the generated Schema.
```

Comment:
Maybe worth a comment here about `./build` to make it easier to tell where this comes from?

Reply:
Commented.

Reply:
Hmm, don't see it in the latest diff. Forgot a `git add` perhaps?